.terraform.lock.hcl

```diff
@@ -0,0 +1,59 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
do we want this committed?

Reply:
Thats a great question, it boils down to 2 things.

1. How we want to react to new versions.
2. Increases the validity of the providers.

I am partial to committing it, because of #1 and the determinism in the version/sha explicit declaration.

more discussion [here](https://discuss.hashicorp.com/t/what-is-the-best-practise-with-terraform-lock-hcl-in-modules-commit-or-not/28648)